packages/cli/src/controllers/auth.controller.ts

```diff
@@ -155,6 +156,11 @@ export class AuthController {
 	@Get('/resolve-signup-token')
 	async resolveSignupToken(req: UserRequest.ResolveSignUp) {
 		const { inviterId, inviteeId } = req.query;
+		const isWithinUsersQuota = Container.get(License).isWithinUsersLimit();
+
+		if (!isWithinUsersQuota) {
+			throw new AuthError('Maximum number of users reached');
```

Comment:
Why `AuthError` instead of `BadRequestError` as above? And let's use the constant?